public/js/components/Tabs.js

```diff
@@ -5,28 +5,46 @@ const { connect } = require("react-redux");
 const actions = require("../actions");
 const { bindActionCreators } = require("redux");
 const { getTabs } = require("../selectors");
-const { debugTab } = require("../client");
+const { debugTab } = require("../clients/firefox");
 
 require("./Tabs.css");
 const dom = React.DOM;
 
-function Tabs({ tabs, actions: boundActions }) {
-  function onSelect(tab) {
-    debugTab(tab.toJS(), boundActions);
+function renderTab(tab, boundActions) {
+  function onSelect(selectedTab) {
+    selectedTab = selectedTab.get("firefox") || selectedTab.get("chrome");
+    debugTab(selectedTab, boundActions);
+  }
+
+  return dom.li(
+    { "className": "tab",
+      "key": tab.get("id"),
+      "onClick": () => onSelect(tab) },
+    dom.div({ className: "tab-title" }, tab.get("title")),
+    dom.div({ className: "tab-url" }, tab.get("url"))
+  );
+}
+
+function renderTabs(tabTitle, tabs, boundActions) {
+  if (tabs.count() == 0) {
+    return null;
   }
 
-  return dom.ul(
-    { className: "tabs" },
-    tabs.valueSeq().map(tab => {
-      return dom.li(
-        { "className": "tab",
-          "data-actor-id": tab.get("actor"),
-          "key": tab.get("actor"),
-          "onClick": () => onSelect(tab) },
-        dom.div({ className: "tab-title" }, tab.get("title")),
-        dom.div({ className: "tab-url" }, tab.get("url"))
-      );
-    })
+  return dom.div({ className: "tab-group" },
+    dom.div({ className: "tab-group-title" }, tabTitle),
+    dom.ul({ className: "tab-list" },
+      tabs.valueSeq().map(tab => renderTab(tab, boundActions))
+    )
+  );
+}
+
+function Tabs({ tabs, actions: boundActions }) {
+  const firefoxTabs = tabs.valueSeq().filter(tab => tab.get("firefox"));
+  const chromeTabs = tabs.valueSeq().filter(tab => tab.get("chrome"));
+
+  return dom.div({ className: "tabs" },
+    renderTabs("Firefox Tabs", firefoxTabs, boundActions),
+    renderTabs("Chrome Tabs", chromeTabs, boundActions)
```

Comment:
I thought we decided to avoid this for now? I'd rather create a query string that specific the target we should debug. It could take either a `chromePort` or `firefoxPort` option and depending on that it would connect to the specific target. It just feels easier to think about what I'm debugging, because I'm debugging a specific "tab" but is it a Firefox or a Chrome tab? We aren't building a toolbox/IDE right now, so I'd prefer to make it clear by using URL params to designate the debuggee.